internal/manifest/manifest.go

```diff
@@ -39,6 +39,7 @@ type Path struct {
 	FinalSHA256 string   `json:"final_sha256,omitempty"`
 	Size        uint64   `json:"size,omitempty"`
 	Link        string   `json:"link,omitempty"`
+	HardLinkId  uint64   `json:"hard_link_id,omitempty"`
```

Comment:
This is not tested anywhere. When adding something to a package, especially something that is user facing such as a field here, it is always a smell that there is no test that references it (in this case manifest_test.go).

Additionally, I think the field name might be too long, please consult the meeting notes that we had with some alternatives.

Another concern that I have is that there are no changes to the `Validate` or `fastValidate` functions.

Reply:
Rafid helped with the tests and validation in [3a9a338](https://github.com/letFunny/chisel/pull/20/commits/3a9a338352f4fbc279e646d87089704d452777c8)